metadata.rb

```diff
@@ -16,6 +16,8 @@
 depends 'compat_resource', '>= 12.14.6'
 depends 'zypper'
 
+suggests 'nssm'
```

Comment:
Hm. How would you define a dependency that is needed for one operating system, but not another? Using depends seems like would include unneeded dependencies on some platforms.